model/flow/dkg.go

```diff
@@ -31,51 +31,64 @@ func (state DKGEndState) String() string {
+// Accordingly, the lower-level cryptographic implementation of the threshold signature and DKG
+// works with these indices. The lower-level cryptographic interface requires that the indices are exactly
+// the set {0, 1, ..., n-1}.
+//
 // On the protocol level, only consensus nodes (identified by their nodeIDs) are allowed to contribute
-// random beacon signature shares. Hence, the protocol level needs to map nodeIDs to DKG indices when
+// random beacon signature shares. Hence, the protocol level needs to map nodeIDs to the indices when
 // calling into the lower-level cryptographic primitives.
 //
 // Formal specification:
-//   - DKGIndexMap completely describes the DKG committee. If there were n parties authorized to participate
+//   - DKGIndexMap completely describes the DKG committee. If n parties are authorized to participate
 //     in the DKG, DKGIndexMap must contain exactly n elements, i.e. n = len(DKGIndexMap)
-//   - The values in DKGIndexMap must form the set {0, 1, …, n-1}.
+//   - The values in DKGIndexMap must form the set {0, 1, …, n-1}, as required by the low level cryptography
+//     module.
 //
-// CAUTION: It is important to cleanly differentiate between the consensus committee 𝒞, the random beacon
-// committee ℛ and the DKG committee 𝒟:
+// CAUTION: It is important to cleanly differentiate between the consensus committee 𝒞, the DKG committee 𝒟
+// and the committee ℛ:
 //   - For an epoch, the consensus committee 𝒞 contains all nodes that are authorized to vote for blocks. Authority
 //     to vote (i.e. membership in the consensus committee) is irrevocably granted for an epoch (though, honest nodes
 //     will reject votes and proposals from ejected nodes; nevertheless, ejected nodes formally remain members of
 //     the consensus committee).
-//   - Only consensus nodes are allowed to contribute to the random beacon. We define the random beacon committee ℛ
-//     as the subset of the consensus nodes, which _successfully_ completed the DKG. Hence, ℛ ⊆ 𝒞.
-//   - Lastly, there is the DKG committee 𝒟, which is the set of parties that were authorized to
-//     participate in the DKG. Mathematically, the DKGIndexMap is an injective function
+//   - The DKG committee 𝒟 is the set of parties that were authorized to
+//     participate in the DKG. Mathematically, the DKGIndexMap is an bijective function
 //     DKGIndexMap: 𝒟 ↦ {0,1,…,n-1}.
+//   - Only consensus nodes are allowed to contribute to the random beacon. We define the committee ℛ
+//     as the subset of the consensus committee (ℛ ⊆ 𝒞), which _successfully_ completed the DKG,
+//     i.e which completed participation in DKG (ℛ ⊆ 𝒟) with a private key that matches the public key assigned to them by
+//     the DKG committee (or a trusted dealer). Nodes in ℛ are therefore able to submit valid random beacon votes.
+//     Based on this definition we note that ℛ ⊆ (𝒟 ∩ 𝒞).
```

Comment:
I think it might help the clarity if we are more concrete about  what it means to "successfully completed the DKG" (happy path and also unhappy path)

```suggestion
//   - Only consensus nodes are allowed to contribute to the random beacon. Informally, we define ℛ as the
//     subset of the consensus committee (ℛ ⊆ 𝒞), which _successfully_ completed the DKG (hence ℛ ⊆ 𝒟).
//     Specifically, r ∈ ℛ iff and only if r has a private Random Beacon key share matching the respective public
//     key share in the `EpochCommit` event. In other words, consensus nodes are in ℛ iff and only if they are able
//     to submit valid random beacon votes. Based on this definition we note that ℛ ⊆ (𝒟 ∩ 𝒞).
```